README.md

```diff
@@ -28,7 +28,7 @@
 
 <h3 align="left">Connect with me:</h3>
 <p align="left">
-<a href="https://linkedin.com/in/https://www.linkedin.com/in/aartithakur" target="blank"><img align="center" src="https://raw.githubusercontent.com/rahuldkjain/github-profile-readme-generator/master/src/images/icons/Social/linked-in-alt.svg" alt="https://www.linkedin.com/in/aartithakur" height="30" width="40" /></a>
```

Comment:
Thanks alot @anmol098 will do the corrections

Reply:
Corrections is done.

Reply:
you could have merged this PR